docs/core/project-sdk/msbuild-props.md

```diff
@@ -524,58 +524,147 @@ The `EnableDefaultNoneItems` property controls whether `None` items (files that
+| Value | Introduced | Meaning |
 |-|-|
-| `Default` | Default mode, where certain rules are enabled as build warnings, certain rules are enabled as Visual Studio IDE suggestions, and the remainder are disabled. |
-| `AllEnabledByDefault` | Aggressive or opt-out mode, where all rules are enabled by default as build warnings. You can selectively [opt out](../../fundamentals/code-analysis/configuration-options.md) of individual rules to disable them. |
-| `AllDisabledByDefault` | Conservative or opt-in mode, where all rules are disabled by default. You can selectively [opt into](../../fundamentals/code-analysis/configuration-options.md) individual rules to enable them. |
+| `All` | .NET 6 | Aggressive or opt-out mode, where all rules are enabled by default as build warnings. You can selectively [opt out](../../fundamentals/code-analysis/configuration-options.md) of individual rules to disable them.<br />This value replaces `AllEnabledByDefault` in .NET 6. |
+| `Default` | .NET 5 | Default mode, where certain rules are enabled as build warnings, certain rules are enabled as Visual Studio IDE suggestions, and the remainder are disabled. |
+| `Minimum` | .NET 6 | More aggressive mode than the `Default` mode. Certain suggestions that are highly recommended for build enforcement are enabled as build warnings. |
+| `None` | .NET 6 | Conservative or opt-in mode, where all rules are disabled by default. You can selectively [opt into](../../fundamentals/code-analysis/configuration-options.md) individual rules to enable them.<br />This value replaces `AllDisabledByDefault` in .NET 6. |
+| `Recommended` | .NET 6 | More aggressive mode than the `Minimum` mode, where more rules are enabled as build warnings. |
+| `AllEnabledByDefault` (deprecated) | .NET 5 | Aggressive or opt-out mode, where all rules are enabled by default as build warnings. You can selectively [opt out](../../fundamentals/code-analysis/configuration-options.md) of individual rules to disable them. |
+| `AllDisabledByDefault` (deprecated) | .NET 5 | Conservative or opt-in mode, where all rules are disabled by default. You can selectively [opt into](../../fundamentals/code-analysis/configuration-options.md) individual rules to enable them. |
 
 > [!NOTE]
-> This property has no effect on code analysis in projects that don't reference a [project SDK](overview.md), for example, legacy .NET Framework projects that reference the Microsoft.CodeAnalysis.NetAnalyzers NuGet package.
+>
+> - In .NET 5 and earlier versions, this property only affects [code-quality (CAXXXX) rules](../../fundamentals/code-analysis/quality-rules/index.md). Starting in .NET 6, if [EnforceCodeStyleInBuild](#enforcecodestyleinbuild) is set to `true`, this property also affects [code-style (IDEXXXX) rules](../../fundamentals/code-analysis/style-rules/index.md).
+> - If you use a compound value for [AnalysisLevel](#analysislevel), for example, `<AnalysisLevel>5-recommended</AnalysisLevel>`, you can omit this property entirely.
```

Comment:
Q: Does a compound value for AnalysisMode have precedence over AnalysisLevel? E.g. if both are specified at different levels.

Reply:
If you specify both then whatever is in `AnalysisLevel` take precedence over `AnalysisMode`.